.github/workflows/nightly-ecs-examples-validator.yml

```diff
@@ -1,61 +1,21 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
 name: Nighly ECS example validator
 on:
   workflow_dispatch:
 
 jobs:
   get-go-version:
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: ./test/acceptance
-    outputs:
-      go-version: ${{ steps.get-go-version.outputs.go-version }}
-    steps:
-    - name: Checkout
-      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-    - name: Determine Go version
-      id: get-go-version
-      run: |
-        echo "Building with Go $(cat .go-version)"
-        echo "go-version=$(cat .go-version)" >> "$GITHUB_OUTPUT"
+    uses: ./.github/workflows/reusable-get-go-version.yml
   go-fmt-and-lint-acceptance:
-    runs-on: ubuntu-latest
     needs:
-      - get-go-version
-    defaults:
-      run:
-        working-directory: ./test/acceptance
-    steps:
-    - name: Checkout
-      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-    - name: Setup Go
-      uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
-      with:
-        go-version: ${{ needs.get-go-version.outputs.go-version }}
-        cache-dependency-path: ./test/acceptance/go.sum
-    - name: Go CI lint
-      uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
-      with:
-        args: "--verbose --enable gofmt"
-        only-new-issues: false
-        skip-pkg-cache: true
-        skip-build-cache: true
-        working-directory: ./test/acceptance
-    - name: Lint Consul retry
-      run: |
-        go install github.com/hashicorp/lint-consul-retry@v1.3.0
-        lint-consul-retry
+    - get-go-version
```

Comment:
NIT: Shoud this have an indent?

Reply:
No it does not since it's either way an array element